project/standard/static/karma.conf.continuous-test.js

```diff
@@ -0,0 +1,116 @@
+var DefinePlugin = require("webpack/lib/DefinePlugin");
+var path = require("path");
+
+module.exports = function karmaConfig(config) {
```

Comment:
in standard projects most of the configurations are the same. Can we share more code for webpack, karma and so on?